2-mandatory/4-people-i-know.js

```diff
@@ -402,16 +419,27 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+friends.forEach(e => {
+  for (const iterator of e.colleagues) {
+    iterator.skills.includes("Multi-tasking")
+      ? colleaguesWhoCanMultitask.push(iterator.name)
+      : null;
+  }
+});
```

Comment:
Good - there are few different ways of doing each of these last 2 questions, so if you have time, review the solution or your fellow trainees' solutions next week.